packages/volto/locales/ca/LC_MESSAGES/volto.po

```diff
@@ -17,6 +17,11 @@ msgstr ""
 "Domain: plone\n"
 "X-Is-Fallback-For: ca-ad ca-es ca-fr ca-it\n"
 
+#. Default: "End Date"
+#: components/manage/Contents/Contents
+msgid " N"
+msgstr ""
```

Comment:
Please don't make changes to the locales that are unrelated to the main purpose of the pull request.

Reply:
alright sorry will keep in mind!